src/test/java/regions/RegionEntitiesTest.java

```diff
@@ -92,20 +90,18 @@ public void testNearestGet() {
 
     @Test
     public void testIsLoaded() {
-        assertEquals(true, regionEntitiesComponent.cellIsLoaded(new Vector2i(0, 0)));
-        assertEquals(true, regionEntitiesComponent.cellIsLoaded(new Vector2i(16, 0)));
-        assertEquals(true, regionEntitiesComponent.cellIsLoaded(new Vector2i(0, -16)));
-        assertEquals(true, regionEntitiesComponent.cellIsLoaded(new Vector2i(24, -25)));
-        assertEquals(false, regionEntitiesComponent.cellIsLoaded(new Vector2i(64, -64)));
-        assertEquals(false, regionEntitiesComponent.cellIsLoaded(new Vector2i(48, -35)));
-        assertEquals(false, regionEntitiesComponent.cellIsLoaded(new Vector2i(10, -36)));
+        assertTrue(regionEntityManager.cellIsLoaded(new Vector2i(0, 0)));
+        assertTrue(regionEntityManager.cellIsLoaded(new Vector2i(16, 0)));
+        assertTrue(regionEntityManager.cellIsLoaded(new Vector2i(0, -25)));
+        assertFalse(regionEntityManager.cellIsLoaded(new Vector2i(98, -124)));
+        assertFalse(regionEntityManager.cellIsLoaded(new Vector2i(351, 234)));
+        assertFalse(regionEntityManager.cellIsLoaded(new Vector2i(153, -134)));
     }
 
     @Test
-    public void testGetRegionssInCell() {
+    public void testGetRegionsInCell() {
         List<EntityRef> testList = new ArrayList<>();
-        Collections.addAll(testList, test);
-        assertEquals(testList, regionEntitiesComponent.getRegionsInCell(new Vector2i(0, 0)));
+        testList.addAll(Arrays.asList(test).subList(0, 5));
```

Comment:
Why not run all tests? Also, consistency with the previous ParcelList changes should be kept. Why not just use collections here as well?

Reply:
The testList here is the regions contained in the cell with at (0,0). Only the first 5 test cases satisfy this, so the array returned by `getRegionsInCell` should only contain those 5 points. 

Not sure I understand your second question about collections though. Could you please explain?